plugins/postcss-custom-properties/src/get-custom-properties-from-root.ts

```diff
@@ -28,29 +27,21 @@ export default function getCustomPropertiesFromRoot(root: Root): Map<string, val
 			return;
 		}
 
-		if (isHtmlRule(rule)) {
-			rule.each((decl) => {
-				if (decl.type !== 'decl') {
-					return;
-				}
-
-				if (!decl.variable || isDeclarationIgnored(decl)) {
-					return;
-				}
-
-				if (decl.value.toLowerCase().trim() === 'initial') {
-					return;
-				}
+		rule.selectors.forEach((selector) => {
+			let specificity = -1;
+
+			if (HTML_WHERE_SELECTOR_REGEXP.test(selector) || ROOT_WHERE_SELECTOR_REGEXP.test(selector)) {
+				specificity = 0;
+			} else if (HTML_SELECTOR_REGEXP.test(selector)) {
+				specificity = 1;
+			} else if (ROOT_SELECTOR_REGEXP.test(selector)) {
+				specificity = 2;
+			} else {
+				return;
+			}
 
-				const thisCascadeLayer = cascadeLayerNumberForNode(decl, cascadeLayersOrder);
```

Comment:
`cascadeLayerNumberForNode(decl, cascadeLayersOrder);`

This looks at the immediate parent, in this case a rule with a selector is the parent of the `decl`, it will never find a layer rule.

This typo was introduced when updating all three `postcss-custom-*` plugins. The other two are for at-rules where the immediate parent can be a layer rule.

Changed to : ``cascadeLayerNumberForNode(rule, cascadeLayersOrder);``